terasoluna-gfw-dependencies/terasoluna-gfw-jodatime-dependencies/pom.xml

```diff
@@ -36,5 +36,11 @@
       <!-- this dependency does not pull servlet-api -->
     </dependency>
     <!-- == End Joda-Time == -->
+    <!-- == Begin Jackson == -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-joda</artifactId>
+    </dependency>
+    <!-- == End Jackson == -->
```

Comment:
- [ ] Please change commit message and title to `move dependency jackson-datatype-joda to gfw-jodatime-dependencies`. And squash 1 commit.

Reply:
fix the commit message and title as pointed out.
8284701